src/features/subscription/NotificationsConnectionModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { AuthenticationButton } from 'features/auth/components/AuthenticationButton/AuthenticationButton'
+import { StepperOrigin } from 'features/navigation/RootNavigator/types'
+import { analytics } from 'libs/analytics'
+import { ButtonWithLinearGradient } from 'ui/components/buttons/buttonWithLinearGradient/ButtonWithLinearGradient'
+import { AppModalWithIllustration } from 'ui/components/modals/AppModalWithIllustration'
+import { InternalTouchableLink } from 'ui/components/touchableLink/InternalTouchableLink'
+import { BicolorUserNotification } from 'ui/svg/icons/BicolorUserNotification'
+import { Spacer, Typo } from 'ui/theme'
+
+interface Props {
+  visible: boolean
+  dismissModal: () => void
+}
+
+export const NotificationsConnectionModal: FunctionComponent<Props> = ({
+  visible,
+  dismissModal,
+}) => {
+  return (
+    <AppModalWithIllustration
+      visible={visible}
+      title="Identifie-toi pour t’abonner à un thème"
+      // rightIconAccessibilityLabel="Ne pas s’identifier"
+      Illustration={StyledIcon}
+      hideModal={dismissModal}>
+      <InformationText>
+        Ton compte te permettra de recevoir toutes les offres et actus en lien avec tes thèmes
+        préférés&nbsp;!
+      </InformationText>
+      <Spacer.Column numberOfSpaces={6} />
+      <ButtonContainer>
+        <InternalTouchableLink
+          as={ButtonWithLinearGradient}
+          wording="Créer un compte"
+          navigateTo={{ screen: 'SignupForm', params: { from: StepperOrigin.FAVORITE } }}
```

Comment:
Il faut ajouter un nouvel élément dans l'enum `StepperOrigin`. Ce paramètre `from` est utilisé pour trigger l'événement StepperDisplayed, qui permet à la data de suivre nos funnels d'inscription / de connexion
```suggestion
          navigateTo={{ screen: 'SignupForm', params: { from: StepperOrigin.SUBSCRIBE_BUTTON } }}
```